src/darwin/Framework/CHIP/MTRDeviceController_Internal.h

```diff
@@ -71,7 +71,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 // queue used to serialize all work performed by the MTRDeviceController
```

Comment:
The comments for the removed property could be removed as well.

Reply:
Good catch, removing.